SingularityService/src/main/java/com/hubspot/singularity/data/NotificationsManager.java

```diff
@@ -39,21 +40,49 @@ public NotificationsManager(
         );
   }
 
+  public void unsubscribe(String email) {
+    removeFromAllowlist(email);
```

Comment:
originally used a conditional based on the `optInEmailMode` but updated to perform both operations so that not much is lost if we revert back to opt-out, but I also couldn't find many documents on how expensive this approach is

Reply:
:+1: - agree with the approach.

Reply:
Should not be bad - if we decide to revert - the block list should still contain all the original information.